### PR TITLE
teuthology: Create tkill group and grant tkill_users right to run `kill`

### PR DESCRIPTION
**Pull request by djgalloway**
To be merged with https://github.com/ceph/ceph-sepia-secrets/pull/145

Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

**Comment by djgalloway** [comment]
To be merged with https://github.com/ceph/ceph-sepia-secrets/pull/145

File: roles/teuthology/tasks/setup_users.yml [hunk 1]
```diff
@@ -8,6 +8,34 @@
...
+  with_items: "{{ tkill_users }}"
+  tags:
+    - user
+  when: tkill_users is defined and tkill_users|length > 0
+
+- name: Grant tkill_users sudo access to /bin/kill
+  lineinfile:
+    dest: /etc/sudoers.d/cephlab_sudo
+    regexp: "^%tkill"
+    line: "%tkill ALL=NOPASSWD: /bin/kill, /usr/bin/kill"
```
Comment (dmick) [inline roles/teuthology/tasks/setup_users.yml]: Assuming this line works as expected, the rest of the ansible looks right to me

Comment (djgalloway) [inline roles/teuthology/tasks/setup_users.yml]: L35 validates syntax

Comment (dmick) [inline roles/teuthology/tasks/setup_users.yml]: Yeah, I saw that; I really meant "works", not "syntax-checks".  but I assume you tested it; I'm just saying "sudoers:  O_o"

**Comment by zmc** [comment]
Any objections to using a more descriptive group name than 'tkill' ?

**Comment by djgalloway** [comment]
`name: thekillers` :)

No objection... whatever you'd prefer

File: roles/teuthology/tasks/setup_users.yml (hunk 1 above)
Comment (djgalloway) [inline roles/teuthology/tasks/setup_users.yml]: I tested on the new teuthology VM as one of the users in the `tkill` group by manually `sudo kill`ing a process owned by another user.  Difficult to test using actual `teuthology-kill`

**Comment by zmc** [comment]
I'd propose "test_admins" because:

1. it's a more self-explanatory name
2. we may want to grant more rights to users in that group in the future

**Comment by djgalloway** [comment]
Renamed unix group to test-admins and ansible var to `test_admins`.  Just tested successfully on teuthology-new.front

**Pull request merged**